examples/smithonsonian/DwC raw/DwC_occurrence_10.csv

```diff
@@ -1,21 +1,21 @@
+http://n2t.net/ark:/65665/3000745d7-3acf-4023-8de0-30a94e187672	PhysicalObject		http://grbio.org/cool/142r-0w94	USNM	Entomology	MaterialSample	AO2MG29	http://n2t.net/ark:/65665/3000745d7-3acf-4023-8de0-30a94e187672	USNMENT1640286.1.16106752	One leg removed for genetic sampling while on loan to GUELPH.		O. Flint	1	Female	Adult	Frozen	in collection			MN344956		15	15	2004	1	15						New Zealand, Southland, [Not Stated], South Island				South Island	New Zealand	Southland	[Not Stated]	Southland, Cascade Stream, Cascade Road, 13 air km NW Riverton		73.0	73.0							-46.26	167.91					Flint, O. S.			Procordulia smithii	Animalia, Arthropoda, Insecta, Odonata, Anisoptera, Corduliidae, Corduliinae	Animalia	Arthropoda	Insecta	Odonata	Corduliidae	Procordulia		smithii			(White)	DNA, RNA, Proteins; Whole genomic DNA			2004-1-15	DNA					USNMENT1640286.1
+http://n2t.net/ark:/65665/300075a27-7020-4325-a598-ef40e7298f9f	PhysicalObject		http://grbio.org/cool/142r-0w94	USNM	Invertebrate Zoology	MaterialSample	AK8EV38	http://n2t.net/ark:/65665/300075a27-7020-4325-a598-ef40e7298f9f	1565936.14804212	Order: 10091; Box Number: MBARI_0125: Box Position: F/2; MBARI Note: SIO Box 95	A2370-C5	R. Haymon et al.	1				in collection					111	111	1991	4	21			AL-2370			North Pacific Ocean		North Pacific Ocean						East Pacific Rise at 9°North					2525.0		9 47 42 N	104 16 54 W		9.795	-104.282									Animalia, Mollusca, Bivalvia, Vesicomyidae	Animalia	Mollusca	Bivalvia		Vesicomyidae							Tissue & Parts; Foot				tissue					1565936
+http://n2t.net/ark:/65665/3000768ed-fc68-446b-aa6d-f8bd437c796a	PhysicalObject		http://grbio.org/cool/142r-0w94	US	Botany	MaterialSample	AE0OW35	http://n2t.net/ark:/65665/3000768ed-fc68-446b-aa6d-f8bd437c796a	12758488		2403	Tibet-MacArthur, J. Wen, Z. Nie, L. Xie, Y. Niu, G. Li, F. Yang, S. Lutz & J. Van De Veire	1				in collection					168	168	2009	6	17	17 Jun 2009					Asia-Temperate, China, Yunnan, Deqin Xian	Asia-Temperate				China	Yunnan	Deqin Xian	Baima Shan		4350.0								28.3419	99.0661								Rhododendron hippophaeoides	Plantae, Dicotyledonae, Ericales, Ericaceae	Plantae		Dicotyledonae	Ericales	Ericaceae	Rhododendron		hippophaeoides			Balf. f. & W.W. Sm.	Tissue & Parts; Leaf; Silica Gel				tissue					
```

Comment:
Regenerated this file using pandas to glue the data from multiple Darwin Core files into one csv.